extractor/src/main/scala/com/digitalasset/extractor/Extractor.scala

```diff
@@ -259,7 +259,12 @@ class Extractor[T](config: ExtractorConfig, target: T)(
       LedgerClientConfiguration(
         applicationId = config.appId,
         ledgerIdRequirement = LedgerIdRequirement.none,
-        commandClient = CommandClientConfiguration(1, 1, java.time.Duration.ofSeconds(20L)),
+        commandClient = CommandClientConfiguration(
+          maxCommandsInFlight = 1,
+          maxParallelSubmissions = 1,
+          defaultDeduplicationTime = java.time.Duration.ofSeconds(20L),
+          submissionIdPropagationSupported = false,
```

Comment:
Does anyone have an idea if I can do it?

Reply:
We won't need this change if we assume that submission ID propagation is supported by all ledgers.

Reply:
Correct.